chainer_chemistry/dataset/preprocessors/common.py

```diff
@@ -118,7 +118,7 @@ def construct_adj_matrix(mol, out_size=-1, self_connection=True):
     return adj_array
 
 
-def construct_discrete_edge_matrix(mol, out_size=-1):
+def construct_discrete_edge_matrix(mol, out_size=-1, self_connection=False):
```

Comment:
can you add docstring for `self_connection`?

self_connection is added as "new" type of label at last channel.